rpm/droid-version.spec

```diff
@@ -0,0 +1,8 @@
+%define rpm_device f5121
```

Comment:
This should be named droid-hal-version-f5121.spec like it was before, right?

Reply:
No, because then it will break existing f5121 package on OBS due to multiple .spec files